### PR TITLE
Add login view and ng content for layouts

### PR DESCRIPTION
**Pull request by JuanGutierrez43**
## Summary 

There was a needed to use a ng-content for the use of the register and login layouts, so I created the layouts component, to use the ng-content over there. Also, I created the login view for this

This is only the use of the ng-content and the view for the login. In the nexts PRs I will create the feature to switch between classes, and change between the login and registrer. 

**New features**

- Login view 
- Use of <ng-content> for the new component 
- New component (layouts) Layouts will manage the both components of the index, the login and the registrer

## Screenshots 
Screenshot from 2020-05-04 18-14-15
Screenshot from 2020-05-05 18-21-51

## Cards

https://trello.com/c/DaGvB9Qq/21-ngcontent

### CODE REVIEW

File: wolox-books/src/app/app.component.html [hunk 1]
```diff
@@ -1 +1 @@
-<app-register-layout></app-register-layout>
+<app-register-layout></app-register-layout>
```
Comment (eledwinn) [inline wolox-books/src/app/app.component.html]: Add empty line here please 😄

File: wolox-books/src/app/login-layout/login-layout.component.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  fields = [
+    {
+      name: 'Email',
+      type: 'text',
+      id: 'email'
+    },
+    {
+      name: 'Password',
+      type: 'text',
```
Comment (eledwinn) [inline wolox-books/src/app/login-layout/login-layout.component.ts]: Did you mean `type: 'password'`?

**Review by eledwinn — CHANGES_REQUESTED** [review]

File: wolox-books/src/app/layouts/layouts.component.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="layout-container">
+    <div class="layout-inputs">
```
Comment (CrisVega08) [inline wolox-books/src/app/layouts/layouts.component.html]: check indentation

File: wolox-books/src/app/layouts/layouts.component.spec.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (CrisVega08) [inline wolox-books/src/app/layouts/layouts.component.spec.ts]: delete this file

File: wolox-books/src/app/login-layout/login-layout.component.html [hunk 5]
```diff
@@ -0,0 +1,8 @@
+      <form class="form-layout">
```
Comment (CrisVega08) [inline wolox-books/src/app/login-layout/login-layout.component.html]: check indentation

File: wolox-books/src/app/login-layout/login-layout.component.spec.ts [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (CrisVega08) [inline wolox-books/src/app/login-layout/login-layout.component.spec.ts]: delete

File: wolox-books/src/app/login-layout/login-layout.component.ts [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login-layout',
+  templateUrl: './login-layout.component.html',
+  styleUrls: ['./login-layout.component.scss']
+})
+export class LoginLayoutComponent implements OnInit {
+
+  fields = [
```
Comment (CrisVega08) [inline wolox-books/src/app/login-layout/login-layout.component.ts]: move a constants.ts file

File: wolox-books/src/app/register-layout/register-layout.component.html [hunk 8]
```diff
@@ -1,17 +1,10 @@
...
-  <div class="login-container">
-    <div class="login-inputs">
-      <img class="logo-wolox-login" src="../../assets/LogoWolox-Original.png"  alt="Wolox Logotype" />
-      <div class="form-container">
-        <form class="form-login">
-          <div class="form-group" *ngFor="let field of fields">
-            <label class="form-label" [for]="field.id">{{field.name}}</label>
-            <input class="default-rectangle form-input" [type]="field.type" [id]="field.id" [name]="field.name" />
+<app-layouts>
+      <form class="form-layout">
```
Comment (CrisVega08) [inline wolox-books/src/app/register-layout/register-layout.component.html]: check indentation

File: wolox-books/src/app/register-layout/register-layout.component.html [hunk 9]
```diff
@@ -1,17 +1,10 @@
...
-          <button class="default-rectangle form-button-empty" type="button">Login</button>
-        </form>
-      </div>
-    </div>
-  </div>
-</body>
+        <button class="default-rectangle form-button-green" type="submit">Sing Up</button>
+        <button class="default-rectangle form-button-empty" type="button">Login</button>
+      </form>
+</app-layouts>
```
Comment (CrisVega08) [inline wolox-books/src/app/register-layout/register-layout.component.html]: add empty line

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

File: wolox-books/src/app/layouts/layouts.component.html [hunk 10]
```diff
@@ -0,0 +1,8 @@
+<div class="layout-container">
+  <div class="layout-inputs">
+    <img class="logo-wolox-layout" src="../../assets/LogoWolox-Original.png" alt="Wolox Logotype" />
```
Comment (CrisVega08) [inline wolox-books/src/app/layouts/layouts.component.html:3]: I suggest create a absolute path for assets

**Review by CrisVega08 — APPROVED** [review]

**Review by eledwinn — APPROVED** [review]

**Pull request merged**